datastream-server-restli/src/main/java/com/linkedin/datastream/server/DatastreamServer.java

```diff
@@ -128,6 +129,7 @@
    *  </ul>
    */
   public DatastreamServer(Properties properties) throws DatastreamException {
+    _properties = properties;
     LOG.info("Start to initialize DatastreamServer. Properties: " + properties);
```

Comment:
Use _properties instead in the entire constructor. It makes sense to do so as it is declared as final.